packages/cli/extensions/environment.js

```diff
@@ -6,15 +6,17 @@ extendEnvironment((hre) => {
   }
 
   hre.cli = {
-    contractName: null,
+    contractFullyQualifiedName: null,
+    contractDeployedAddress: null,
```

Comment:
This change is so the CLI is not dependant on the deployment artifact, and just can be configured with an address to use. This change was necessary so the flow can become agnostic between Satellites and Modules.